.github/workflows/ci.yml

```diff
@@ -49,6 +49,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install
 
+      - name: Build (ignoring failures)
+        run: pnpm run build || true
```

Comment:
Explaining: some newly enabled lint rules require type information. I'm under the impression `types` won't be fully filled out (i.e. some packages are seen as `any` / not-existing in editors) until `pnpm run build` happens. Is that the case?

Reply:
Yeah `pnpm run build` is needed to generate the types. I'd be fine with this since our CI also builds packages. I think this can be refactored further later to share the turbo cache in a later PR.